server/src/test/java/io/unitycatalog/server/base/catalog/BaseCatalogCRUDTest.java

```diff
@@ -1,19 +1,20 @@
 package io.unitycatalog.server.base.catalog;
 
 import static io.unitycatalog.server.utils.TestUtils.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.tuple;
 import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
can you please also switch this and the assertNotNull check to AssertJ here and in other files?

Reply:
Updated with [`94d582b`](https://github.com/unitycatalog/unitycatalog/pull/279/commits/94d582b0aae31c2b3e4807d8b092ddd2acb56be6) and [`699dd19`](https://github.com/unitycatalog/unitycatalog/pull/279/commits/699dd190e1adc0ab28909999c8876a5e48aea732). Please let me know if further changes need to be made.